app/src/webkit/java/org/mozilla/focus/webkit/WebkitView.java

```diff
@@ -23,6 +22,8 @@
 import android.webkit.WebView;
 import android.webkit.WebViewDatabase;
 
+import androidx.annotation.NonNull;
```

Comment:
do we need this?

Reply:
Just a re-sort to follow the alphabetical order.